src/v8/api-generator-dev-dependencies.ts

```diff
@@ -0,0 +1,17 @@
+import { existsSync } from "fs";
+
+import { PackageJson } from "../util/package-json.util";
+
+export const stage = "before-install";
+
+/**
+ * Add @comet/api-generator package to dev dependencies of api
+ */
+export default async function updatePrettier() {
+    if (existsSync("api/package.json")) {
+        console.log("🚀 api/package.json add @comet/api-generator-package to dev-dependencies");
+        const packageJson = new PackageJson("api/package.json");
+        packageJson.addDependency("@comet/api-generator", "^8.0.0", true);
```

Comment:
we should install the latest possible version to make sure all comet packages have the same version. and since https://github.com/vivid-planet/comet-starter/pull/690 we pin the version, should be done here as well

Reply:
i think that's doable, but not with the helper available right now. I can try :-)

Reply:
due to we are in a before-install stage, we probably could work with:

`npm view @comet/admin@8 version --json` and pick the latest one. @nsams what do you think or do you have any better idea?

Reply:
@nsams added the mentioned behaviour and showcased with version 7.

https://github.com/user-attachments/assets/88d0372b-0e7b-4d93-81d9-c9b5d2208545

Reply:
we already have this: https://github.com/vivid-planet/comet-upgrade/blob/main/src/util/get-latest-package-version.ts

used here https://github.com/vivid-planet/comet-upgrade/blob/main/src/index.ts#L43

always the latest isn't correct I'm assuming (when there is already a v9 but you update to v8 you should install the latest v8)